Void/Void.Core/Commands.fs

```diff
@@ -3,10 +3,13 @@
 [<RequireQualifiedAccess>]
 type Command =
     | Noop
+    | PublishEvent of Event
+    //
+    | ChangeToMode of Mode

```

Comment:
What would you think about the name `NextNode`? `ChangeToNode` sounds like an action not a type constructor.